### PR TITLE
NuGet.Server update

### PR DESCRIPTION
**Pull request by maartenba**
Ready for review.

@yishaigalatzer @csharpfritz @joyhui  @xavierdecoster

### CODE REVIEW

**Comment by xavierdecoster** [comment]
Looks good to me :shipit:

**Comment by csharpfritz** [comment]
I think this works...   :shipit:

File: NuGet.Docs/Create/Hosting-Your-Own-NuGet-Feeds.md [hunk 1]
```diff
@@ -65,31 +65,32 @@ You can also host a remote (or internal) feed on a server that runs IIS. There a
 1. NuGet.Server
 2. NuGet Gallery
 
-For relatively small projects with a small set of packages go with NuGet.Server, it is basically a view of a network share or local folder through http, and as such is easy to setup and works quite well when the number of packages is small. _However_ if the package count is high, say in the thousands, the server will take a very long time to boot. The alternative is to use the [NuGet Gallery Project](https://github.com/NuGet/NuGetGallery/wiki/Hosting-the-NuGet-Gallery-Locally-in-IIS) it is more complex to set up and host, but offers a lot more nuget.org like features.
+For relatively small projects with a small set of packages go with NuGet.Server, it is basically a view of a network share or local folder through http, and as such is easy to setup and works quite well when the number of packages is relatively small.
```
Comment (yishaigalatzer) [inline NuGet.Docs/Create/Hosting-Your-Own-NuGet-Feeds.md]: with the new nuget.server this is not relevant. Can we add a blurb describing it?

**Comment by yishaigalatzer** [comment]
:watch:

File: NuGet.Docs/Create/Hosting-Your-Own-NuGet-Feeds.md (hunk 1 above)
Comment (maartenba) [inline NuGet.Docs/Create/Hosting-Your-Own-NuGet-Feeds.md]: How about a change to this:

> NuGet.Server is ideal when a simple NuGet server is desired. It is basically a view of a network share or local folder through http(s), and as such is easy to setup. The NuGet Gallery provides user management and additional features such as an extensive web UI that allows searching and exploring packages from within the browser.

**Comment by maartenba** [comment]
Should we cross-link to our blog post as well? https://github.com/NuGet/nuget.github.com/pull/97

**Comment by yishaigalatzer** [comment]
yes

**Comment by maartenba** [comment]
Ok will add once the blog has :shipit:

**Comment by maartenba** [comment]
Updated the blurb. Please check again :)

**Comment by maartenba** [comment]
This has been released last week. Can we merge this one?

**Comment by csharpfritz** [comment]
Thanks for following up on this

**Pull request merged**